app/assets/javascripts/application.js

```diff
@@ -234,6 +240,7 @@ function template_info(div, url) {
     },
     error: function(jqXHR, textStatus, errorThrown) {
       $(div).html('<div class="alert alert-warning alert-dismissable">' +
+        '<span class="pficon-layered"><span class="pficon pficon-warning-triangle"></span><span class="pficon pficon-warning-exclamation"></span></span>' +
```

Comment:
We can even use the same one, but just need to add kind to it so we will be able to use pficon and fa, not only glyph. should i add this function in this patch also?
return '<span class="#{kind}-' + name + '" />';